website/content/docs/templates/json_to_hcl.mdx

```diff
@@ -0,0 +1,159 @@
+---
+description: |
+  This section highlights the differences between JSON and HCL2 templates and explains the parallels between the specifications.
+page_title: JSON to HCL
+---
+
+# Packer templates
+
+Packer uses templates to orchestrate builds for one or more images.
+In [legacy JSON templates](/packer/docs/templates/legacy_json_templates), you would declare a series of builders, provisioners and post-processors to build images.
+In [HCL2 templates](/packer/docs/templates/hcl_templates), things are different, as the configuration language allows you to specify builders through sources, and weave them in build blocks.
+
+This document aims to explain the parallels between the two configuration files, and what you
+should expect when moving a template away from JSON to HCL2.
+
+In addition to this document, you may find the `packer hcl2_upgrade` command useful to help you convert your existing JSON templates to HCL2.
+See the [hcl2_upgrade](/packer/docs/commands/hcl2_upgrade) page for more information on its usage.
+
+## Builders
+
+Builders are components that are specialized for a platform. Their role is to set the stage for
+the steps that will let you build the final image that you want to build with Packer.
```

Comment:
I'm not sure about the `build the final image artifact` phrasing, since I would assume users may see artifacts as a side-effect of building their images, which is essentially what they use Packer for.